service/src/main/kotlin/fi/espoo/evaka/placement/PlacementController.kt

```diff
@@ -178,17 +178,25 @@ class PlacementController(
         user: AuthenticatedUser,
         @PathVariable("placementId") placementId: PlacementId
     ) {
-        Audit.PlacementCancel.log(targetId = placementId)
         accessControl.requirePermissionFor(user, Action.Placement.DELETE, placementId)
 
         db.connect { dbc ->
             dbc.transaction { tx ->
-                val (childId, startDate, endDate) = tx.cancelPlacement(placementId)
-                asyncJobRunner.plan(
-                    tx,
-                    listOf(AsyncJob.GenerateFinanceDecisions.forChild(childId, DateRange(startDate, endDate)))
-                )
+                tx.cancelPlacement(placementId).let {
```

Comment:
You could use `also` instead of `let` and remove `it` from the end of the block 🤓

Reply:
Fixed 🤓 🤓 🤓